Client/U2FExtensions.swift

```diff
@@ -243,6 +243,29 @@ class U2FExtensions: NSObject {
             ]
             makeCredentialRequest.options = makeOptions
             
+            let exclusionList: [Any] = publicKey.excludeCredentials.compactMap({
```

Comment:
Why is this type set to `[Any]` instead of `[YKFFIDO2PublicKeyCredentialDescriptor]` (or even just letting it be inferred)?

Reply:
It can't be inferred for some odd reason Xcode cannot determine what the return type of the closure is. I left it as `[Any]` because that's what the library's parameter is.. However I'll change it for this case.